services/galley/Procfile

```diff
@@ -1 +1 @@
-web: LOG_LEVEL=Info ./dist/galley -p $PORT --brig-host=$BRIG_WEB_HOST --brig-port=$BRIG_WEB_PORT --gundeck-host=$GUNDECK_WEB_HOST --gundeck-port=$GUNDECK_WEB_PORT --cassandra-host=$GALLEY_CASSANDRA_HOST --cassandra-port=$GALLEY_CASSANDRA_PORT --cassandra-keyspace $GALLEY_CASSANDRA_KEYSPACE --team-events-queue-name $GALLEY_SQS_TEAM_EVENTS --aws-region $KHAN_REGION
+web: LOG_LEVEL=Info ./dist/galley -p $PORT --brig-host=$BRIG_WEB_HOST --brig-port=$BRIG_WEB_PORT --gundeck-host=$GUNDECK_WEB_HOST --gundeck-port=$GUNDECK_WEB_PORT --cassandra-host=$GALLEY_CASSANDRA_HOST --cassandra-port=$GALLEY_CASSANDRA_PORT --cassandra-keyspace $GALLEY_CASSANDRA_KEYSPACE --team-events-queue-name $GALLEY_SQS_TEAM_EVENTS --aws-region $KHAN_REGION --fake-sqs-host $FAKE_SQS_HOST --fake-sqs-port $FAKE_SQS_PORT
```

Comment:
You could add a line here:
```
sqs: ./fake-sqs.sh
```
which would start `fake-sqs` together with `galley`

Reply:
I tried that; it doesn't work due to timing issues, i.e. it will start galley and start attempting to connect to SQS before SQS is up and the queue created. Adding a `sleep` before starting galley also doesn't work, as it then shows errors about `/i/monitoring` returning a 500. It seems keiretsu is quite eager. 
I propose we investigate using some other form of scheduling for all of the dependencies (cassandra, elasticsearch, redis, and now fake-sqs, ...) whether with docker compose or other means. That is not directly related to this PR though.